src/js/constants.js

```diff
@@ -21,7 +21,6 @@ module.exports = {
   title: 'Mozilla Observatory',
   urls: {
     api: 'https://http-observatory.security.mozilla.org/api/v1/',
-    ssh: 'https://sshscan.rubidus.com/api/v1/',
```

Comment:
This makes me wonder: can we link to some other SSH scan tool for people who have been using Observatory for SSH scanning?

Reply:
We could link users directly to the CLI tool here? https://github.com/mozilla/ssh_scan